cmake/helpers/CheckDependentLibraries.cmake

```diff
@@ -173,114 +186,79 @@ endif ()
 
 find_package(PROJ 6.0 REQUIRED)
 
-find_package(TIFF 4.0)
-if (TIFF_FOUND)
-  set(HAVE_TIFF ON)
-  set(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES} ${TIFF_INCLUDE_DIR})
-  set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} ${TIFF_LIBRARIES})
-  set(GDAL_USE_LIBTIFF_INTERNAL
-      OFF
-      CACHE BOOL "")
-endif ()
-if (NOT HAVE_TIFF)
-  set(GDAL_USE_LIBTIFF_INTERNAL
-      ON
-      CACHE BOOL "")
-endif ()
+gdal_check_package(TIFF "Support for the Tag Image File Format (TIFF)." VERSION 4.0 CAN_DISABLE)
 set_package_properties(
   TIFF PROPERTIES
   URL "https://libtiff.gitlab.io/libtiff/"
-  DESCRIPTION "support for the Tag Image File Format (TIFF)."
+  DESCRIPTION "Support for the Tag Image File Format (TIFF)."
   TYPE RECOMMENDED)
+invert_on_off(GDAL_USE_TIFF NOT_GDAL_USE_TIFF)
+set(GDAL_USE_LIBTIFF_INTERNAL
```

Comment:
I would prefer if the variable was named `GDAL_USE_TIFF_INTERNAL`, similar to `GDAL_USE_TIFF`.

Reply:
done